### PR TITLE
Fix duplicated glossary entry for CRI

### PR DESCRIPTION
**Pull request by shurup**
Currently, we have two conflicting glossary entries for CRI:
* `content/en/docs/reference/glossary/cri.md` (added in 2019 via #13024)
* `content/en/docs/reference/glossary/container-runtime-interface.md` (added in 2021 via #30606)

If you open the [actual glossary](https://kubernetes.io/docs/reference/glossary/?all=true), the first one will be displayed only under the name "Container runtime interface (CRI)".

This PR:
* removes the current `cri.md` in favour of `container-runtime-interface.md` (since the latter was added/reviewed more recently);
* renames current `container-runtime-interface.md` file to `cri.md` (because this is how it's currently done for `cni` and `csi`);
* makes a few technical changes to a new `cri.md`: its `term_id` becomes `cri`, its title becomes `Container Runtime Interface (CRI)` (includes the abbreviation), its `tags` changed from `cri` to `fundamental` (that's how it was for the currently displayed CRI entry);
* substitutes all existing `<glossary_tooltip term_id="container-runtime-interface"...>` entries to the new `term_id` (i.e. `cri`).

### CODE REVIEW

File: content/en/docs/reference/glossary/cri.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+title: Container Runtime Interface (CRI)
 id: cri
-date: 2019-03-07
-full_link: /docs/concepts/architecture/#container-runtime
+date: 2021-11-24
+full_link: /docs/concepts/architecture/cri
 short_description: >
-    An API for container runtimes to integrate with kubelet
-
+  The main protocol for the communication between the kubelet and Container Runtime.
```
Comment (sftim) [inline content/en/docs/reference/glossary/cri.md:7]:
```suggestion
  Protocol for communication between the kubelet and the local container runtime.
```
Short is good here.

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by shurup** [comment]
Thanks, @sftim! I applied your suggestion and squashed it. A new `lgtm` is needed, though :sweat_smile:

**Pull request merged**